docs/frontend_development.md

```diff
@@ -57,6 +57,23 @@ $ nvm use
 $ script/develop
 ```
 
+The first build will take around a minute on a recent machine.
```

Comment:
We shouldn't add this in the middle of the instructions for development. I don't even know if we should include this at all honestly. Just mention the error message in your PR/issue and people will find it with Google.